src/testing/state_machine.zig

```diff
@@ -109,58 +165,56 @@ pub fn StateMachineType(
                 .reserved, .root => unreachable,
                 .register => return 0,
                 .echo => {
+                    const thing = state_machine.forest.grooves.things.get(123);
+                    const key: u64 = if (thing) |t| t.timestamp else timestamp;
+
+                    state_machine.forest.grooves.things.put(&.{
+                        .timestamp = key,
+                        .id = 123,
+                        .value = @truncate(u64, vsr.checksum(input)),
+                    });
```

Comment:
As the variables are now used, their `_ = variable` discard statements can be removed.

Reply:
Good catch; updated.